test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Builder/FunctionConfigurationTest.cs

```diff
@@ -830,6 +830,27 @@ public void GetEdmModel_SetsDateTimeAsParameterType_WorksForDefaultConverter()
             Assert.True(parameter.Type.IsNullable);
         }
 
+        [Fact]
+        public void GetEdmModel_SetDecimalAsParameterType()
```

Comment:
>GetEdmModel_SetDecimalAsParameterType [](start = 20, length = 37)

I wanted to make sure that I am not missing something here. Is this relevant to the change?

Reply:
You are right. I remove this changes.